src/utils/generator/ReplicaFiller.cpp

```diff
@@ -141,15 +141,15 @@ void ReplicaFiller::init() {
 	std::shared_ptr<Object> object = std::make_shared<ObjectShifter>(_object, _origin);
 	basisContainer.setBoundingBox(object);
 
-	std::list<ChemicalPotential> lmu;
+
 #ifdef ENABLE_MPI
 	DomainDecomposition domainDecomp;
 #else
 	DomainDecompBase domainDecomp;
 #endif
 	Domain domain(0, nullptr);
 	_inputReader->readPhaseSpaceHeader(&domain, 0.0);
-	_inputReader->readPhaseSpace(&basisContainer, &lmu, &domain, &domainDecomp);
+    _inputReader->readPhaseSpace(&basisContainer, &domain, &domainDecomp);
```

Comment:
spaces -> tabs + `Class* ptr`

Reply:
Changed as requested: 183c219

Just read the style guide and i think judging from the code base, I'm not the only one who didn't know we had one.